packages/jetpack-affiliate/lib/Jetpack_Affiliate.php

```diff
@@ -68,7 +76,8 @@ public function get_affiliate_code() {
 	 * @return string The passed URL with the code added.
 	 */
 	public function add_code_as_query_arg( $url ) {
-		if ( '' !== ( $aff = $this->get_affiliate_code() ) ) {
+		$aff = $this->get_affiliate_code();
+		if ( '' !== $aff ) {
 			$url = add_query_arg( 'aff', $aff, $url );
 		}
 		return $url;
```

Comment:
dd_action( 'init', array( 'Jetpack_Affiliate', 'init' ) ); Should that be updated to include the namespace?

Reply:
good catch, fixed